mitosheet/css/elements/Toggle.css

```diff
@@ -0,0 +1,71 @@
+.toggle-label {
+    --toggle-label-width: 60px;
+    --toggle-toggle-width: 24px;
+    --toggle-toggle-margin: 3px;
+
+    position: relative;
+    display: inline-block;
+    width: var(--toggle-label-width);
+    height: 30px;
+}
+
+/* Don't display the default html input so we can created out own custom styled one */
+.toggle-label input {display: none;}
+  
```

Comment:
Make sure to test this boi on windows before requesting review; this looks somewhat fancy and might have compatibility issues.